src/core/SimpleReport.tsx

```diff
@@ -1,51 +1,75 @@
 import * as React from 'react';
-import {Button, Col, DropdownItem, DropdownMenu, DropdownToggle, Row, Table, UncontrolledButtonDropdown} from 'reactstrap'
-import { useTable, useSortBy, usePagination, TableOptions, TableInstance, Column, SortByFn } from "react-table";
+
+import {Button, Col, Row, DropdownItem, DropdownMenu, DropdownToggle, Table, UncontrolledButtonDropdown, Input, Label, Form, Popover, PopoverBody, PopoverHeader, UncontrolledPopover, ButtonGroup, FormGroup, Card} from 'reactstrap'
+import { useTable, useSortBy, usePagination, Row as RowRT, TableOptions, TableInstance, useFilters, Column, DefaultFilterTypes, FilterType, useGlobalFilter, PluginHook } from "react-table";
+
+
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 import {
 	faSortUp,
 	faSortDown,
 } from "@fortawesome/free-solid-svg-icons";
+
+import { ReactNode } from 'react';
+
 import { TableOptionsCbi, TableInstanceCbi, TableStateCbi, TableColumnOptionsCbi } from 'react-table-config';
+import { DisplayableProps } from 'util/StringifyObjectProps';
+
+export type SimpleReportColumn = {
+	accessor: string,
+	Header: ReactNode,
+	disableSortBy?: boolean,
+	width?: number,
+	toggleHidden?: boolean
+}
 
 type SimpleReportRequiredProps<T_Data> = {
 	keyField: keyof T_Data,
-	data: T_Data[],
+	data: T_Data[] | DisplayableProps<T_Data[]>,
```

Comment:
data should be the raw unformatted data, so that we can sort on numbers/dates etc properly.  Formatting should be specified by providing a renderer to the Cell attribute of the column object